public/docs/_examples/toh-6/ts/app/app.module.ts

```diff
@@ -12,7 +12,7 @@ import { HttpModule }    from '@angular/http';
 
 // #enddocregion v1
 // Imports for loading & configuring the in-memory web api
-import { InMemoryWebApiModule } from 'angular-in-memory-web-api';
+import { InMemoryWebApiModule } from 'angular-in-memory-web-api/in-memory-web-api.module';
```

Comment:
Ok. It should be noted as a caveat to rollup if this is not the way it has to be done normally unless we figure out a way to use the barrel export.